src/conventions-and-definitions.md

```diff
@@ -14,9 +14,9 @@ The `Curve25519` function, which multiplies a curve (or twist) point by a "scala
 secret key), is defined in the paper. The `clamp_scalar` function is defined as follows:
 
     fn clamp_scalar(scalar: [u8; 32]) -> [u8; 32] {
-        scalar[0] &= 248
-        scalar[31] &= 127
-        scalar[31] |= 64
+        scalar[0] &= 248 // TODO paper show {0, 8, 16, 24, . . . , 248}, I understand 7 different values, something like 1 << (salar[0] % 7) 
```

Comment:
This forces the bottom 3 bits to zero which gives 0, 8, 16...

Reply:
yes :facepalm: